ipfs/ipfsobject.nim

```diff
@@ -0,0 +1,10 @@
+import pkg/libp2p
+
+type
+  IpfsObject* = object
```

Comment:
Lets get rid of the IPFS naming.

Reply:
Calling it simply `Object` felt weird, but should work. Or would you prefer to call it `DaggerObject`?

Reply:
Yes, as we agreed, let's drop the ipfs moniker to avoid confusion.

Reply:
Addressed in #6.